ingestion/src/metadata/utils/time_utils.py

```diff
@@ -120,7 +120,7 @@ def convert_timestamp(timestamp: str) -> Union[int, float]:
 
 
 @deprecated("Use `datetime_to_timestamp` instead", "1.7.0")
-def convert_timestamp_to_milliseconds(timestamp: Union[int, float]) -> int:
+def datetime_to_timestamp(timestamp: Union[int, float]) -> int:
```

Comment:
Hey, if you check this module, there's already a function called `datetime_to_timestamp`.

The issue was not about renaming this function but to use the new one since it has a different logic.
You could delete this function and rely on the other one. (You should still check if no other change is needed to make it work that way)

Reply:
Hello Ice, I see
I should fix it in another branch, so I'm revertsing it.